Template-2.md

```diff
@@ -0,0 +1,249 @@
+     2. Numbered sub-item 2
+
+  > **Note:** Markdown supports only **two** indention levels for numbered lists.
+
+## Code Blocks
+
+For code blocks, enter a Return, three backticks (```) before and after the code block, plus another Return:
+
+```Text within three backticks style code or command line samples```
+
+## Code Comments
+
+<!--For example, this code denotes a comment, and information written inside of it will not appear on the website but can be used as a reference for others viewing the file.-->
+
+## Images
+
+To insert an image (graphic) into your Page, first upload the image file to the **/img/** folder in the appropriate GitHub repository.  Next, at the image insertion point in your Page, add this format to link to the image:
+
+![This is what I want a screen reader to say for 508 compliance]({{site.baseurl}}/img/thisisthesampleimagefile.png)
+
+  > To align an image in your Page, use the **align** instruction at the end of an image link (i.e., **"left"**, **"center"**, or **"right"**) using this format:
+
+![Alt text for an image goes here]({{site.baseurl}}/folder that image is in/imagename.png){:align="right"}
+
+## Alerts, Warnings, and Information Blocks
+
+Use the Alert, Warning, or Information Block formatting to highlight specific information in your Page.
+
+### Warning blocks
+
+> This text will appear as a **warning flag**, which is a yellow banner on the website. (The ">" symbol and the coded line directly underneath the text will create the formatting for this flag.) Warning flags can be used for notifications, such as when a user should skip a certain procedure.
+{:class="warning"}
+
+### Alert blocks
+
+> This text will appear as an **alert flag**, which is a red banner on the website. Alert flags may be used for notifications, such as common problems that may occur.
+{:class="alert"}
+
+### Information blocks
+
+> This text will appear as **informational flag**, which is a green banner on the website. Information flags can be used for notifications such as useful links or helpful tips.
+{:class="info"}
+
+## Links to References or Other Information
+
+To link to useful references, information, or another page (especially that may be needed to complete the procedures), use this link format: 
+
+[This is what I want my link to say]({{site.baseurl}}/insertlink/) 
+
+  > **Note:**  This is the **end of the Markdown Template**.
+
+# Writing and Style Resources
+
+For the FICAM Guides, general writing guidelines include:
+
+  * All pages should be brief.
+  * Use titles to help the user identify "jumping off points" for information.
+  * Paragraphs should be short. 
+  * All text should be written in plain language as much as possible. 
+  * Use numbered steps, bullet lists, and graphics whenever possible. 
+
+The following references provide additional information about plain language, writing, and style:
+
+  * [18F Content Guide]({{site.baseurl}}/https://content-guide.18f.gov/)
```

Comment:
These links need to be updated.  None will resolve - they have the site.baseurl in front of the https.   External links should use the target -> blank (open in new window).